org.mfat.systemdpilot/org.mfat.systemdpilot.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.mfat.systemdpilot
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: systemd-pilot
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --system-talk-name=org.freedesktop.systemd1
+  - --system-talk-name=org.freedesktop.login1
+  - --socket=session-bus
+  - --system-talk-name=org.freedesktop.PolicyKit1
+  - --socket=system-bus
+  - --filesystem=xdg-run/gvfsd
+  - --filesystem=xdg-config/gtk-4.0:ro
+  - --filesystem=xdg-config/gtk-3.0:ro
+modules:
+  - name: systemd-pilot
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/python3/dist-packages/systemd_pilot
+      - cp src/main.py /app/lib/python3/dist-packages/systemd_pilot/
+      - mkdir -p /app/bin
+      - echo '#!/usr/bin/env python3' > /app/bin/systemd-pilot
+      - echo 'import sys' >> /app/bin/systemd-pilot
+      - echo 'sys.path.append("/app/lib/python3/dist-packages")' >> /app/bin/systemd-pilot
+      - echo 'from systemd_pilot.main import app' >> /app/bin/systemd-pilot
+      - echo 'app.run(None)' >> /app/bin/systemd-pilot
```

Comment:
don't `echo` things, have an actual executable or a script

Reply:
also, why does this not support an install system? Why are you manually copying and moving files around?